LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2022 Laluka
+
+Permission to use, copy, modify, and distribute this software for any purpose
```

Comment:
Let's modify (or change licence?) to forbin comercial use, it's for individual work, helpers, educational purpose, we don't want no pay-to-use shiny frontend! :dagger: